int16 degree
---
bool ok
